### PR TITLE
[FrameworkBundle] Fix deprecation message for booting a kernel twice

### PR DESCRIPTION
**Pull request by jschaedl**
| Q             | A
| ------------- | ---
| Branch?       | 4.4
| Bug fix?      | no
| New feature?  | no 
| Deprecations? | no 
| Tickets       | - 
| License       | MIT
| Doc PR        | -

### CODE REVIEW

File: src/Symfony/Bundle/FrameworkBundle/Test/WebTestCase.php [hunk 1]
```diff
@@ -42,7 +42,7 @@ private function doTearDown()
     protected static function createClient(array $options = [], array $server = [])
     {
         if (static::$booted) {
-            @trigger_error(sprintf('Calling "%s()" while a kernel has been booted is deprecated since Symfony 4.4 and will throw in 5.0, ensure the kernel is shut down before calling the method.', __METHOD__), E_USER_DEPRECATED);
+            @trigger_error(sprintf('Calling "%s()" while a kernel has been booted is deprecated since Symfony 4.4 and will throw an Exception in 5.0, ensure the kernel is shut down before calling the method.', __METHOD__), E_USER_DEPRECATED);
```
Comment (Tobion) [inline src/Symfony/Bundle/FrameworkBundle/Test/WebTestCase.php]: exception should not be written with a capital letter

**Comment by nicolas-grekas** [comment]
`src/Symfony/Bundle/SecurityBundle/DependencyInjection/Security/*Factory.php` use the same pattern, we should fix them all.

**Comment by jschaedl** [comment]
@nicolas-grekas 

> src/Symfony/Bundle/SecurityBundle/DependencyInjection/Security/*Factory.php use the same pattern, we should fix them all.

I fixed them all and found another one in here: `src/Symfony/Component/Security/Core/Authentication/Provider/LdapBindAuthenticationProvider.php`

**Review by fabpot — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @jschaedl.

**Pull request merged**